source/manual/schemas.html.md

```diff
@@ -0,0 +1,41 @@
+# Schemas used on GOV.UK
+Here are a list of schema's being used
+
+### [HowTo][how-to]
+
+[how-to]: https://schema.org/HowTo
+
+Used in step-by-step, for example, [learn-to-drive-a-car][learn-to-drive-a-car]
```

Comment:
The text in the first set of square brackets is what will be displayed on the page, so it would be good to make it a human-readable title, like "Learn to Drive a Car" or something similar.

Reply:
This also applies to the other links on the page.